### PR TITLE
update zh-tw translation for v0.13

### PR DESCRIPTION
**Pull request by Lafudoci**
@erciccione @0140454 Would you like to give a review?

### CODE REVIEW

File: translations/monero-core_zh-tw.ts [hunk 1]
```diff
@@ -1079,235 +1079,242 @@ The following information will be deleted
...
         <location filename="../pages/settings/SettingsInfo.qml" line="188"/>
         <source>Cancel</source>
-        <translation type="unfinished">取消</translation>
+        <translation>取消</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="202"/>
         <source>Invalid restore height specified. Must be a number.</source>
-        <translation type="unfinished"></translation>
+        <translation>無效的回復高度。 必須要是個數字。</translation>
```
Comment (0140454) [inline translations/monero-core_zh-tw.ts]: 句號後的空白是不是可以不用？

Comment (Lafudoci) [reply translations/monero-core_zh-tw.ts]: 已修改為逗號較為通順。

**Comment by Lafudoci** [comment]
@0140454 Thanks for you help.

>  Use 建立 instead of 創建

Revised in all translation.
> Use 重新命名 instead of 易名

Revised.
> Insert space between English and Chinese characters
        For example
            輸入指令後按下 enter 送出 (範例: help 或 status)
            唯讀錢包 (view-only wallet)

Revised.
> Use 「...」 instead of "..." (aka. &quot;)

Revised. But it might need someone to check the encoding format.

File: translations/monero-core_zh-tw.ts [hunk 2]
```diff
@@ -1079,235 +1079,242 @@ The following information will be deleted
...
         <location filename="../pages/settings/SettingsNode.qml" line="232"/>
         <source>Uses a third-party server to connect to the Monero network. Less secure, but easier on your computer.</source>
-        <translation type="unfinished"></translation>
+        <translation>使用第三方的遠端節點以連接到 Monero 網路，較不安全，但簡易使用。</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsNode.qml" line="277"/>
         <source>To find a remote node, type &apos;Monero remote node&apos; into your favorite search engine. Please ensure the node is run by a trusted third-party.</source>
-        <translation type="unfinished"></translation>
+        <translation>如果需要遠端節點，上網搜尋 「 Monero 遠端節點 」。請確認該節點是由可信任的第三方所架設。</translation>
```
Comment (Lafudoci) [inline translations/monero-core_zh-tw.ts]: @erciccione Could you please check if the directly using `「` and `」` here is technically OK with encoding?

Comment (erciccione) [reply translations/monero-core_zh-tw.ts]: It is :)

Comment (0140454) [reply translations/monero-core_zh-tw.ts]: There is a space between ```尋``` and ```「```.
I think it can be removed.

**Comment by Lafudoci** [comment]
Squashed all commits. Thank you @0140454 and @erciccione .

**Pull request merged**